2025-09-18 21:22:55,993 INFO Starting keno_rng TUI
2025-09-18 21:22:55,996 INFO Extension not built or import failed: No module named 'search_rng_module'. Attempting to build...
2025-09-18 21:22:57,466 ERROR Failed to build or import extension.
Traceback (most recent call last):
  File "/storage/emulated/0/Download/Keno_rng/main.py", line 37, in ensure_extension_built
    importlib.import_module(EXT_NAME)
  File "/data/data/com.termux/files/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1324, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'search_rng_module'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/storage/emulated/0/Download/Keno_rng/main.py", line 47, in ensure_extension_built
    importlib.import_module(EXT_NAME)
  File "/data/data/com.termux/files/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 921, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 813, in module_from_spec
  File "<frozen importlib._bootstrap_external>", line 1293, in create_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
ImportError: dlopen failed: library "/storage/emulated/0/Download/Keno_rng/search_rng_module.cpython-312.so" needed or dlopened by "/data/data/com.termux/files/usr/lib/libpython3.12.so.1.0" is not accessible for the namespace "(default)"
2025-09-18 21:23:25,549 INFO Menu selection: ingest
2025-09-18 21:23:45,432 INFO Menu selection: ingest
2025-09-18 21:25:05,762 INFO Saved draw for machine 6th ave pit stop: [1, 2, 4, 5, 7, 9, 13, 18, 22, 31, 34, 41, 44, 46, 48, 53, 55, 65, 67, 73]
2025-09-18 21:25:22,255 INFO Menu selection: scan
2025-09-19 22:12:32,011 INFO Starting Keno Toolkit
2025-09-19 22:20:49,827 INFO Starting Keno Toolkit
2025-09-19 22:23:21,061 INFO Starting Keno Toolkit
2025-09-20 10:28:07,536 INFO Starting Keno Toolkit
2025-09-20 11:04:48,002 INFO Starting Keno Toolkit
2025-09-20 11:10:15,856 INFO Starting Keno Toolkit
2025-09-20 11:15:54,175 INFO Starting Keno Toolkit
